vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/ExperimentInformationPage.java

```diff
@@ -40,20 +49,24 @@
 @PermitAll
 public class ExperimentInformationPage extends Div implements BeforeEnterObserver, RouterLayout {
 
+  private ExperimentItemCollection itemCollection;
   @Serial
   private static final long serialVersionUID = -3443064087502678981L;
   private static final Logger log = LoggerFactory.logger(ProjectViewPage.class);
   private final transient ExperimentInformationPageHandler experimentInformationPageHandler;
+  private final ExperimentInformationService experimentInformationService;
 
   public ExperimentInformationPage(
       @Autowired ProjectNavigationBarComponent projectNavigationBarComponent,
       @Autowired ExperimentDetailsComponent experimentDetailsComponent,
       @Autowired ExperimentListComponent experimentListComponent,
-      @Autowired ProjectInformationService projectInformationService) {
+      @Autowired ProjectInformationService projectInformationService,
+      @Autowired ExperimentInformationService experimentInformationService) {
     Objects.requireNonNull(projectNavigationBarComponent);
     Objects.requireNonNull(experimentDetailsComponent);
     Objects.requireNonNull(experimentListComponent);
     Objects.requireNonNull(projectInformationService);
+    this.experimentInformationService = Objects.requireNonNull(experimentInformationService);
     setupBoard(projectNavigationBarComponent, experimentDetailsComponent, experimentListComponent);
```

Comment:
NIT: I'd also recommend to remove the board component and replace it with a css grid based design to allocate the content and support components. However this can be tackled in a seperate PR

Reply:
that is out of scope of this PR